gobblin-metastore/src/main/java/gobblin/metastore/FsStateStore.java

```diff
@@ -57,24 +58,21 @@
   // Class of the state objects to be put into the store
   private final Class<? extends State> stateClass;
 
-  public FsStateStore(String fsUri, String storeRootDir, Class<? extends State> stateClass)
-      throws IOException {
+  public FsStateStore(String fsUri, String storeRootDir, Class<? extends State> stateClass) throws IOException {
```

Comment:
LinkedIn's coding style template automatically put `throw ...` onto the next line.